tests/phpunit/tests/test-wpml-gutenberg-stings-in-block.php

```diff
@@ -19,7 +20,7 @@ public function it_finds_paragraph() {
 
 		$strings_in_block = new WPML_Gutenberg_Strings_In_Block( $config_option );
 
-		$paragraph = 'some paragraph';
+		$paragraph = 'some paragraph &amp special chars';
```

Comment:
@strategio Should this be `&amp` or `&amp;`?

Reply:
Good catch @brucepearson :)

This is corrected.